conanfile.py

```diff
@@ -241,6 +249,7 @@ def configure_cmake(self):
         self.check_dependencies()
 
         cmake = CMake(self)
+        print('cmake.build_folder', cmake.build_folder)
```

Comment:
Prefer `self.output.info(<msg>)` instead

Reply:
That line should not be there. I forgot to remove it.

Reply:
I removed it. Can you please stop all running travis builds and keep the last one running?

Reply:
done!